hack/gen-swagger-doc/README.md

```diff
@@ -0,0 +1,18 @@
+This folder contains the sources needed to build the gen-swagger-doc container.
+
+To build the container image, 
+
+```
+$ sudo docker build -t gcr.io/google_containers/gen-swagger-docs:v1 .
+```
+
+To generate the html docs,
+
+```
+$ ./run-gen-swagger-docs.sh <API version> <absolute output path, default to PWD>
```

Comment:
I think the general rule is if it has a `docker build` statement, it should probably go in a make file. I didn't mean that you should have written a makefile *instead of* your .sh file, rather in *addition to*. :)